tests/utils/matchers/value-type-asymmetric-matchers.ts

```diff
@@ -18,10 +21,43 @@ expect.extend({
     },
 });
 
+expect.extend({
+    toBeCloseToVec3(actual: unknown, expected: Readonly<Vec3>, numDigits = 5) {
+        return matchers.toStrictEqual.call(
+            this,
+            actual,
+            expect.objectContaining({
+                x: expect.toBeAround(expected.x, numDigits),
+                y: expect.toBeAround(expected.y, numDigits),
+                z: expect.toBeAround(expected.z, numDigits),
```

Comment:
Why close the the same value for all elements?

Reply:
That's OK: if we're expecting two vectors to be equal, we're expecting their corresponding components to be equal.

Reply:
I misunderstood.